scripts/globals/magic.lua

```diff
@@ -447,6 +447,7 @@ end
 -- target has to the given effect (stun, sleep, etc..)
 function getEffectResistance(target, effect)
     local effectres = 0
+    local statusres = target:getMod(tpz.mod.STATUSRES)
     if (effect == tpz.effect.SLEEP_I or effect == tpz.effect.SLEEP_II) then
         effectres = tpz.mod.SLEEPRES
```

Comment:
note to self: come back later and murder this giant if/else tower with great fury.